src/looprestoration.rs

```diff
@@ -404,64 +408,49 @@ unsafe fn wiener_rust<BD: BitDepth>(
         edges,
     );
 
-    let mut hor: [uint16_t; 27300] = [0; 27300];
-    let mut hor_ptr: *mut uint16_t = hor.as_mut_ptr();
-    let filter: *const [int16_t; 8] = ((*params).filter.0).as_ptr();
-    let bitdepth = bd.bitdepth().as_::<libc::c_int>();
-    let round_bits_h = 3 as libc::c_int + (bitdepth == 12) as libc::c_int * 2;
-    let rounding_off_h = (1 as libc::c_int) << round_bits_h - 1;
-    let clip_limit = (1 as libc::c_int) << bitdepth + 1 + 7 - round_bits_h;
-    let mut j = 0;
-    while j < h + 6 {
-        let mut i = 0;
-        while i < w {
-            let mut sum = (1 as libc::c_int) << bitdepth + 6;
+    // Values stored between horizontal and vertical filtering don't
+    // fit in a uint8_t.
+    let mut hor = [0u16; 70 /*(64 + 3 + 3)*/ * REST_UNIT_STRIDE];
```

Comment:
```suggestion
    let mut hor = [0; 70 /*(64 + 3 + 3)*/ * REST_UNIT_STRIDE];
```

Doesn't seem necessary.